app/src/main/res/values/dimens.xml

```diff
@@ -14,4 +14,9 @@
     <dimen name="sales_overview_top_menu_text_size">16sp</dimen>
     <dimen name="viewpager_height">251dp</dimen>
     <dimen name="viewpager_margin">16dp</dimen>
+    <dimen name="viewPager2_height">290dp</dimen>
```

Comment:
Watch out when setting a predefined height to items. It will look odd on low-res screens. Better use `wrap-content`.

Reply:
No because all images don't have the same size.